service/src/main/kotlin/no/nav/su/se/bakover/service/grunnlag/VilkårsvurderingService.kt

```diff
@@ -14,7 +14,15 @@ internal class VilkårsvurderingServiceImpl(
     private val formueVilkårsvurderingRepo: FormueVilkårsvurderingRepo,
 ) : VilkårsvurderingService {
     override fun lagre(behandlingId: UUID, vilkårsvurderinger: Vilkårsvurderinger) {
-        uføreVilkårsvurderingRepo.lagre(behandlingId, vilkårsvurderinger.uføre)
-        formueVilkårsvurderingRepo.lagre(behandlingId, vilkårsvurderinger.formue)
+        when (vilkårsvurderinger) {
+            is Vilkårsvurderinger.Revurdering -> {
+                uføreVilkårsvurderingRepo.lagre(behandlingId, vilkårsvurderinger.uføre)
```

Comment:
Er det innafor å ønse seg transactions her?  😸

Reply:
Ja. Lagring av vilkår og søknadsbehandling som én enhet står på refaktoreringsblokka, naturlig å refaktorere vekk behovet for transaksjon her da.